docs/development.md

```diff
@@ -81,3 +81,18 @@ _TODO Add testing instructions once a testing suite is implemented_
 <!--
 Probably using bloc_test and mockito or mocktail.
 -->
+
+
+## Releasing
+
+Before releasing, make sure to create a new `release/a.b.c` branch from the `dev` branch and implement any pre-release changes. Then run `cz bump`, which should update the version in `cz.yaml` and `pubspec.yaml`.
+
+_TODO: Remove `git-cliff`, check how the procedure above goes with building a release apk build and having to run `pub get` (if a dry run of `cz bump`is not needed first along with manually changing version in `pubspec.yaml` and running `pub get` to update gradle and other build files) and investigate signing tags by PGP._
+
+To release _a signed version_ of the Gibsonify android app via an `apk`, you need to have the Gibsonify java signing key on your machine. This key was generated on 30 January 2022 and has the id `7c909176`, using instructions found [here](https://docs.flutter.dev/deployment/android#signing-the-app) and [here](https://fahadjameel.com/2020/09/07/how-to-properly-sign-your-flutter-apps-before-uploading-to-the-android-app-store/). You then need to have a `key.properties` file in the `android` subdirectory of Gibsonify, which includes the path and password to the key. Only trusted developers should be able to build signed releases of Gibsonify, and this key should only be circulated amongst them.
+
+Having satisfied this, an Android APK can be generated by running `flutter build`, which should default to `flutter build --release`. This will output an `apk` to `[project]/build/app/outputs/apk/release`. This is a fat `apk` file that should run on most Android architectures.
+
+The SHA256 checksum for the `app-release.apk` file should then be generated by running `shasum -a 256 app-release.apk > app-release-checksum.txt` on Mac or `sha256sum release.apk > app-release-checksum.txt` on Linux. These files can then also be signed by the PGP key of the developer making the release.
```

Comment:
In the future, this will be changed to just signing the release file with Gibsonify PGP key.